dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/DataValueSetController.java

```diff
@@ -120,109 +116,98 @@
     // Get
     // -------------------------------------------------------------------------
 
-    @GetMapping( params = { "format", "compression", "attachment" } )
```

Comment:
FYI: This change here is to counter another bug found during testing. Currently the app does not send the `compression` parameter when _Uncompressed_ is selected which then would map by `Accept` header. Since the download is is a generic file download by the browser the `Accept` header will always be some browser default which usually contains `application/xml` wherefore we would always end up in the XML handling method when the `compression` parameter was missing. I figured if the user sends a `format` parameter we do want to map based on that so I removed the other two "requirements" here. This now works as expected.